scargo/commands/check.py

```diff
@@ -186,15 +186,25 @@ def check(self) -> None:
         super().check()
 
     def check_file(self, file_path: Path) -> CheckResult:
+        copyrights = self.copyright_desc.split("\n")[:-1]
+        slice_present = 0
+        copyright_present = False
         with open(file_path, encoding="utf-8") as file:
             for line in file.readlines():
-                if self.copyright_desc in line:
-                    return CheckResult(problems_found=0)
-                if "copyright" in line.lower():
-                    logger.warning(
-                        "Incorrect and not excluded copyright in %s", file_path
-                    )
-                    return CheckResult(problems_found=1, fix=False)
+                for copyrights_slice in copyrights:
```

Comment:
I think `copyright_line` would be a better name for this variable. Slices have a specific meaning in Python and it doesn't fit here.

Reply:
done